### ISSUE TITLE
Learning Rate Configuration

### ISSUE THREAD

**Issue by HannahHaensen** [DeepSceneSeg/EfficientPS#18]
Thanks for providing the code. However I am currently training EfficientPS using the Cityscapes Dataset. I am a bit curious about the MultiStep Learning Rate Scheduler explained in the Paper.

In the paper it is said that a linear warmup from 1/3 * base to base is performed, than 10 epochs with 1e-4 and than the base learning rate decreased by a factor of 10 at every milestone. However I can not find the implementation for the 10 epochs in the code and also the freezing of the iABN layers.

How is your Cityscapes model trained, that you provide?

**Comment by mohan1914** [comment]
To freeze the iABN layers, you can set the require_grad=False in the config file. And then add the extra 10 epochs in the mutli-step parameters and the total epoch. 
However, we don't use this lr schedule for training our network with this repository. Instead, we follow the following additional training protocol:
We start another round of training with the same initial lr, but halve the multi-step iteration steps as well as the total epoch.
(the model of this round is initialized from the previous one)
It brings an additional gain of 0.6% for Cityscapes comparative to 0.1% of the frozen iABN layer scheme.

**Comment by mohan1914** [comment]
To freeze the iABN layers, you can set the require_grad=False in the config file. And then add the extra 10 epochs in the mutli-step parameters and the total epoch.
However, we don't use this lr schedule for training our network with this repository. Instead, we follow the following additional training protocol:
We start another round of training with the same initial lr, but halve the multi-step iteration steps as well as the total epoch.
(the model of this round is initialized from the previous one)
